CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased][unreleased]
+
+### Changed
+
+- Expose `AuthenticationStrength`'s `compliance` number property instead of
+  `strength` string.
+
+### Removed
+
+- Dropped support for Node.js 6.
+- Outdated `inherits()` method (in favor of `util.inherits()` available in
```

Comment:
```suggestion
- Deprecate `inherits()` method (in favor of `util.inherits()` available in
```

Reply:
But it was removed in https://github.com/metarhia/common/commit/2fe0916a0c5f0e0bfa32882a0ef662d4cddc727f, not deprecated.